docs/home/index.md

```diff
@@ -34,3 +34,21 @@ The [Tools](/docs/tools/) page contains a list of native and third-party tools f
 ## Troubleshooting
 
 The [Troubleshooting](/docs/tasks/debug-application-cluster/troubleshooting) page outlines some resources for troubleshooting and finding help.
+
+## Supported Versions
+
+Kubernetes has a _X.Y.Z_ versioning scheme, where _X_ is the major version, _Y_ is the minor version, and _Z_ is the patch version. 
+
+Kubernetes is supported for three minor versions at a time. This includes the current release version and two previous versions. 
+
+See the [Kubernetes Release](https://github.com/kubernetes/kubernetes/releases) page on GitHub for the latest release information.
+
+### Minor Versions
+
+Different components are expected to be compatible across different version skew relative to the master. Nodes may lag master by up to two versions, but not exceed the master version. Clients may lag master by one version and may exceed master up to one version.
+
```

Comment:
I stumbled over the first sentence under Minor Versions. I suggest changing it to something like this:
A certain amount of skew is permissible between master components, node components, and the kubectl client.

Reply:
Thanks for the review @steveperry-53 !
Updated almost as suggested ("A certain amount of version skew...")